pyfolio/bayesian.py

```diff
@@ -165,6 +166,169 @@ def model_returns_t(data, samples=500):
     return trace
 
 
+def model_best(y1, y2, samples=1000):
+    """Bayesian Estimation Supersedes the T-Test
+
+    This model replicates the example used in:
+
+    Kruschke, John. (2012) Bayesian estimation supersedes the t
+    test. Journal of Experimental Psychology: General.
+
+    The original pymc2 implementation was written by Andrew Straw and
+    can be found here: https://github.com/strawlab/best
+
+    Ported to PyMC3 by Thomas Wiecki (c) 2015.
```

Comment:
Is this the same as your function in PyMC3? Either way we should change this line to pyfolio.